api/Performance.json

```diff
@@ -496,6 +559,16 @@
             "safari_ios": {
               "version_added": "11"
             },
+            "samsunginternet_android": [
+              {
+                "version_added": true
+              },
+              {
+                "prefix": "webkit",
```

Comment:
We should remove this support statement; the prefixed version was added in Chrome 25 and removed in Chrome 28, which both equate to Samsung Internet 1.5 (so it was never in a Samsung Internet version).